spec/jobs/user_email_spec.rb

```diff
@@ -288,6 +288,35 @@
       expect(mail.body).to include("asdfasdf")
     end
 
+    context "confirm_new_email" do
+      let(:email_token) { Fabricate(:email_token, user: user) }
+      before do
+        EmailChangeRequest.create(
+          user: user,
+          requested_by: requested_by,
+          new_email_token: email_token,
+          new_email: "testnew@test.com",
+          change_state: EmailChangeRequest.states[:authorizing_new]
+        )
+      end
+
+      context "when the change was requested by admin" do
+        let(:requested_by) { Fabricate(:admin) }
+        it "passes along true for the requested_by_admin param" do
+          UserNotifications.expects(:confirm_new_email).with(user, email_token: email_token.token, requested_by_admin: true)
```

Comment:
Are we able to test this without mocks? We should try and avoid mocks and stubs whenever possible.